packages/dioxus-blitz/src/documents/dioxus_document.rs

```diff
@@ -251,6 +471,18 @@ impl WriteMutations for MutationWriter<'_> {
                 content: value.to_string(),
             });
         }
+
+        let parent = node.parent;
+
+        if let Some(parent) = parent {
+            // if the text is the child of a style element, we want to put the style into the stylesheet cache
+            let parent = self.doc.get_node(parent).unwrap();
+            if let NodeData::Element(ref element) = parent.raw_dom_data {
+                if element.name.local.as_ref() == "style" {
+                    self.doc.add_stylesheet(value);
+                }
+            }
+        }
```

Comment:
To a be a little more robust, I was going to suggest push style tags to a queue when they are encountered. Then processing once the mutations list has been processed. We'll also need to deal with edits to existing stylesheets. And making sure multiple stylesheets are added in the right order ("document order").

This will probably do for now though...